http-server/src/main/java/io/micronaut/http/server/RouteExecutor.java

```diff
@@ -337,7 +337,7 @@ RouteMatch<?> findErrorRoute(Throwable cause,
 
         if (errorRoute != null) {
             if (LOG.isDebugEnabled()) {
-                LOG.debug("Found matching exception handler for exception [{}]: {}", cause.getMessage(), errorRoute);
+                LOG.debug("Found matching exception handler for exception [{}]: {}", cause.getMessage(), errorRoute, cause);
```

Comment:
Here the recipe had added the `cause`, but I'm thinking that's maybe not desired here given that we're handling the error route.
```suggestion
                LOG.debug("Found matching exception handler for exception [{}]: {}", cause.getMessage(), errorRoute);
```